intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -116,14 +116,14 @@ LLM running script args explanations:
 | -p / --prompt     | prompt to start generation with (default: empty)                        |
 | -n / --n_predict  | number of tokens to predict (default: -1, -1 = infinity)                |
 | -t / --threads    | number of threads to use during computation (default: 56)               |
-| -b / --batch_size | batch size for prompt processing (default: 512)                         |
+| -b / --batch_size_truncate | batch size for prompt processing (default: 512)                         |
 | -c / --ctx_size   | size of the prompt context (default: 512, can not be larger than specific model's context window length)                                                                                |
 | -s / --seed       | NG seed (default: -1, use random seed for < 0)                          |
 | --repeat_penalty  | penalize repeat sequence of tokens (default: 1.1, 1.0 = disabled)       |
 | --color           | colorise output to distinguish prompt and user input from generations   |
 | --keep            | number of tokens to keep from the initial prompt (default: 0, -1 = all) |
 | --glm_tokenizer   | the path of the chatglm tokenizer (default: THUDM/chatglm-6b)           |
```

Comment:
should be tokenizer? more general...

Reply:
this tokenizer is specially designed for Chatglm1 by zhenzhong